biopac/c01_bidsify_discovery.py

```diff
@@ -57,7 +57,7 @@
         sub = utils.initialize._extract_bids(filename, 'sub') # 'sub-0056'
         ses = utils.initialize._extract_bids(filename, 'ses') # 'ses-03'
         task = utils.initialize._extract_bids(filename, 'task')
-        logger.info(f"\n\n__________________{sub} {ses} {task}__________________")
+        logger.info("__________________%s %s %s__________________", sub, ses, task)
```

Comment:
Yarik you are wrong! it must have been
```suggestion
        logger.info("__________________%s %d %s__________________", sub, ses, task)
```